app/controllers/application_controller.rb

```diff
@@ -1,9 +1,6 @@
 class ApplicationController < ActionController::Base
   attr_accessor :repo
-
-  def current_user
-    @current_user = repo.user.find_by_id(session[:user_id]) if session[:user_id]
-  end
```

Comment:
`current_user` is used in `admin?` below...I'm going to guess that it's re-defined by `authenticate_user!`?

Reply:
Yup, devise or rails or something comes with a `current_user` method that's much better than this one...

Reply:
awesome